src/components/DisplayCategories.jsx

```diff
@@ -2,33 +2,118 @@ import { getReviews } from "../requests";
 import { useState } from "react";
 import { Link } from "react-router-dom";
 import { useEffect } from "react";
+import moment from "moment";
 
 const DisplayCategories = ({ chosenCategory }) => {
   const [categoryReviews, setCategoryReviews] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [sort, setSort] = useState(null);
+
   useEffect(() => {
     getReviews().then((allReviews) => {
-      setCategoryReviews(
-        allReviews.filter((review) => {
-          return review.category === chosenCategory;
-        }),
-        setLoading(false)
-      );
+      let categoryFilter = allReviews.filter((review) => {
+        return review.category === chosenCategory;
+      });
+      if (sort === "newest-oldest" || sort === null) {
+        setCategoryReviews(
+          [...categoryFilter].sort(
+            (a, b) =>
+              moment(b.created_at).format("YYYYMMDDHHmmss") -
+              moment(a.created_at).format("YYYYMMDDHHmmss")
+          )
+        );
+
+        setLoading(false);
+      }
+      if (sort === "oldest-newest") {
+        setCategoryReviews(
+          [...categoryFilter].sort(
+            (a, b) =>
+              moment(a.created_at).format("YYYYMMDDHHmmss") -
+              moment(b.created_at).format("YYYYMMDDHHmmss")
+          )
+        );
+      }
+      if (sort === "votes_asc") {
+        setCategoryReviews(
+          [...categoryFilter].sort((a, b) => a.votes - b.votes)
+        );
+      }
+      if (sort === "votes_desc") {
+        setCategoryReviews(
+          [...categoryFilter].sort((a, b) => b.votes - a.votes)
+        );
+      }
+      if (sort === "comments_asc") {
+        setCategoryReviews(
+          [...categoryFilter].sort((a, b) => a.comment_count - b.comment_count)
+        );
+      }
+      if (sort === "comments_desc") {
+        setCategoryReviews(
+          [...categoryFilter].sort((a, b) => b.comment_count - a.comment_count)
+        );
+      }
```

Comment:
This looks (and works) really well.  However, we don't want to filter/sort our reviews on the front end.  We want to use the queries we set up for our API ( for examples /reviews?category=apples&sort_by=desc) to do this.

Modify the getReviews function you have in requests.js so that it can take optional sort by and filter queries.  Look up axios params to do this.  Here's a helpful link:
https://masteringjs.io/tutorials/axios/get-query-params